src/snowflake/cli/plugins/object/stage/diff.py

```diff
@@ -154,7 +162,47 @@ def build_md5_map(list_stage_cursor: SnowflakeCursor) -> Dict[str, str]:
     }
 
 
-def stage_diff(local_path: Path, stage_fqn: str) -> DiffResult:
+def _get_relative_paths_to_sync(
+    relative_files_to_sync: List[Path], local_path: Path, remote_paths: Set[str]
```

Comment:
local_path is really a root folder here, right? All paths are expected to be relative to it. I'd name it something like local_root to be clear.